### DISCUSSION TITLE
L10n FR Chorus Account for Odoo 13 is not available ?

### DISCUSSION THREAD

**Discussion by younessomlk** [OCA/odoo-community.org#101] [General]
Hello, 
I need Chorus Account module for Odoo 13 and i'd like to know if it's available somewhere before attempting to upgrad it. 
Currently it's available on 12, 14,15, and 16.

![image](https://github.com/OCA/odoo-community.org/assets/50590196/0894f233-b378-4656-bed6-82a33caf87b0)

https://github.com/OCA/l10n-france/tree/16.0/l10n_fr_chorus_account.
https://github.com/OCA/l10n-france

Thank you

**Comment by legalsylvain** [comment]
Hi. If the module is not available for V13, I think it will not be now ported, as V13 is quite obsolete. I advise you to migrate your database in a recent V16 version. (or port the code and propose a PR). 
Regards.

**Reply by younessomlk** [reply]
Thank you for your reply.
